frontend/components/text-classifier/labeling-rules/RulesAnnotationArea.vue

```diff
@@ -0,0 +1,280 @@
+<!--
+  - coding=utf-8
+  - Copyright 2021-present, the Recognai S.L. team.
+  -
+  - Licensed under the Apache License, Version 2.0 (the "License");
+  - you may not use this file except in compliance with the License.
+  - You may obtain a copy of the License at
+  -
+  -     http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing, software
+  - distributed under the License is distributed on an "AS IS" BASIS,
+  - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  - See the License for the specific language governing permissions and
+  - limitations under the License.
+  -->
+
+<template>
+  <div class="annotation-area">
+    <div v-if="labels.length">
+      <p v-if="!dataset.query.text">Start by entering a query in the search box</p>
+      <p v-else>Select a label for your query</p>
+      <label-search
+        v-if="labels.length > maxVisibleLabels"
+        :search-text="searchText"
+        @input="onSearchLabel"
+      />
+      <ClassifierAnnotationButton
+        v-for="label in visibleLabels"
+        :id="label.class"
+        :key="`${label.class}`"
+        v-model="selectedLabels"
+        :allow-multiple="isMultiLabel"
+        :label="label"
+        :class="[!dataset.query.text ? 'non-reactive' : null, 'label-button']"
+        :data-title="label.class"
+        :value="label.class"
+        @change="updateLabel"
+      >
+      </ClassifierAnnotationButton>
+
+      <a
+        v-if="visibleLabels.length < filteredLabels.length"
+        href="#"
+        class="feedback-interactions__more"
+        @click.prevent="expandLabels()"
+        >+{{ filteredLabels.length - visibleLabels.length }}</a
+      >
+      <a
+        v-else-if="visibleLabels.length > maxVisibleLabels"
+        href="#"
+        class="feedback-interactions__more"
+        @click.prevent="collapseLabels()"
+        >Show less</a
+      >
+    </div>
+    <div v-else class="empty-labels">
+      <p>This doesn't have any labels yet.</p>
+      <p>
+        To create new rules you need al least two labels. It's highly recommended to also annotate some records with these labels. Go to the annotation mode to 
+        <a href="#" @click.prevent="changeToAnnotationViewMode">create the labels and annotate some records</a>.
+      </p>
+    </div>
+    <rule-annotation-area-metrics :metrics="metrics" />
+    <p v-if="currentRule && selectedLabels.includes(currentRule.label) && currentRule.description === description">
+      {{saved ? 'Rule has been saved' : 'You have already a rule saved with that label'}}.
```

Comment:
Loading a rule from Management rules view raise message `You have already a rule saved with that label`. This should be avoided.

Maybe we can improve condition

Reply:
copy changed to "This query is already saved as a rule"